app/javascript/components/fonticon-picker/font-icon-picker-new.jsx

```diff
@@ -0,0 +1,106 @@
+import React, { useState } from 'react';
+import {
```

Comment:
Could you change this file name to font-icon-picker-ddf

Reply:
ok